### ISSUE TITLE
perl-dbd-mysql: security update to 4.0.43

### ISSUE THREAD

**Issue by MingcongBai** [AOSC-Dev/aosc-os-abbs#673]
```
Perl DBD::mysql driver does not enforce SSL/TLS encryption when option
mysql_ssl=1 is enabled. Enabling encryption depends on announcement from
MySQL server what it supports which can man-in-the-middle attack spoof.
DBD::mysql does not enforce SSL/TSL encryption even when certificate is
specified via connection parameter mysql_ssl_ca_file.
```

Fixed CVE: CVE-2017-10789.

**Comment by MingcongBai** [comment]
Fixed with CVE-2017-10789.

**Comment by MingcongBai** [comment]
Fixed with 3175e02f8af19884ba6dde88e4455c0f78cffa48.

**Comment by MingcongBai** [comment]
Use AOSA-2017-0047.
